src/graphql/candyMachine.ts

```diff
@@ -64,11 +64,13 @@ const downloadZip = async (
   if (!dirExists) {
     await mkdirp(processDir);
     await download(zipUrl, zipFile);
-    await exec('/usr/bin/7z x ' + zipFile + ` -y -o${zipFilesDir}`);
+    await exec('7z x ' + zipFile + ` -y -o${zipFilesDir}`);
   } else {
     throw Error('zip dir already exists');
   }
 
+  console.log(`complete zip download: ${zipFilesDir}`)
```

Comment:
should you be using `logger.info` instead of `console.log`?

Reply:
nice. moved to tracking using logger in main handler.